certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2652 @@
+                  "x-ms-visibility": "advanced"
+                },
+                "Id": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Id"
+                },
+                "ModifiedDate": {
+                  "type": "string",
+                  "description": "ModifiedDate",
+                  "x-ms-visibility": "advanced"
+                },
+                "ModifiedById": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ModifiedById",
+                  "x-ms-visibility": "advanced"
+                },
+                "CreatedDate": {
+                  "type": "string",
+                  "description": "CreatedDate",
+                  "x-ms-visibility": "advanced"
+                },
+                "CreatedById": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "CreatedById",
+                  "x-ms-visibility": "advanced"
+                },
+                "Guid": {
+                  "type": "string",
+                  "description": "Guid",
+                  "x-ms-visibility": "advanced"
+                },
+                "IsDeleted": {
+                  "type": "boolean",
+                  "description": "IsDeleted",
+                  "x-ms-visibility": "advanced"
+                }
+              }
+            },
+            "headers": {
+              "accept": {
+                "description": "accept",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Get Ticket",
+        "description": "Get One Ticket",
+        "operationId": "GetOneTicket",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Enter Tikit ID",
+            "x-ms-summary": "Tikit ID"
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Define oData select",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "Define oData select"
+          }
+        ]
+      }
+    },
+    "/FileUpload/{id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get File(s) Attached to a Ticket",
+        "description": "Get File(s) Attached to a Ticket",
+        "operationId": "GetFileAttached",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Tikit ID",
+            "description": "Enter Tikit ID"
+          }
+        ]
+      }
+    },
+    "/Comment": {},

```

Comment:
@TikitTeam Empty operations can be removed as they would not add any value to your connector.